bench.sh

```diff
@@ -18,6 +18,7 @@ export GRPC_CLIENT_CPUS=${GRPC_CLIENT_CPUS:-"1"}
 export GRPC_REQUEST_SCENARIO=${GRPC_REQUEST_SCENARIO:-"complex_proto"}
 export GRPC_IMAGE_NAME="${GRPC_IMAGE_NAME:-grpc_bench}"
 export GRPC_GHZ_TAG="${GRPC_GHZ_TAG:-0.114.0}"
+export GRPC_MEMLOCK_ULIMIT=${GRPC_MEMLOCK_ULIMIT:-"-1:-1"}
```

Comment:
This looks a bit overly generous. Can we limit it to a default I found on my machine?
```suggestion
export GRPC_MEMLOCK_ULIMIT=${GRPC_MEMLOCK_ULIMIT:-"8192:8192"}
```